minos/common/configuration/config.py

```diff
@@ -78,6 +80,8 @@
     "commands.queue.password": "commands_queue_password",
     "commands.broker": "commands_broker",
     "commands.port": "commands_port",
+    "saga.broker": "saga_host",
```

Comment:
In order to follow the same naming convention:

```suggestion
    "saga.broker": "saga_broker",
```

Reply:
Done!